### PR TITLE
DS-270 Maintain Checkbox label position between loading and non-loading states

### PR DESCRIPTION
**Pull request by david-cho-lerat-sonarsource**

### CODE REVIEW

**Review by jeremy-davis-sonarsource — APPROVED** [review]
LGTM!

File: src/components/checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -17,28 +17,27 @@
...
-
-  hasError?: boolean;
+  innerClassName?: string;
   isDisabled?: boolean;
   isLoading?: boolean;
-
+  label?: string;
   onCheck: (checked: boolean | 'indeterminate', id?: string) => void;
   onFocus?: VoidFunction;
+  title?: string;
```
Comment (gregaubert) [inline src/components/checkbox/Checkbox.tsx:40]: @jeremy-davis-sonarsource @david-cho-lerat-sonarsource should we enable an eslint rule for that?

Comment (david-cho-lerat-sonarsource) [reply src/components/checkbox/Checkbox.tsx:40]: I wouldn't mind always having the props in alpha order, at least it's a system. I know the opposing arguments of grouping them "semantically", or "functions together", or "important first", etc, but in real life when no order is enforced none of this is followed in the long run and we just end up with "whatever". I'd much prefer to enforce the alpha order and be done with it.

Comment (jeremy-davis-sonarsource) [reply src/components/checkbox/Checkbox.tsx:40]: Sarath (?) mentioned having the same VScode config for everyone (and every project!), which think makes sense, and will solve this aspect as well

Comment (gregaubert) [reply src/components/checkbox/Checkbox.tsx:40]: I think this is enforced by an eslint rule on SC, not sure the vscode config will help there.
But I guess we also want a common eslint config for all projects.

**Pull request merged**